### DISCUSSION TITLE
Hosting for server-side render and incremental-static-regeneration

### DISCUSSION THREAD

**Discussion by zhex900** [vercel/next.js#34946] [Help]
I am trying to understand where SSR and ISR are computed. For hosting, there must be a nodejs server that does SSR and ISR. 

Services like Vercel or Netlify, how does it carry out SSR? What is the cost? How does it scale?

**Comment by scottklein7** [comment]
I am also wondering that as well! So is Netlify supporting ISR?

**Comment by jrutter** [comment]
Is there a list anywhere of which hosts support ISR?
It sounds like only Vercel and Netlify do it, does AWS or Amplify support it too?
